### DISCUSSION TITLE
Application aapproved and no benefits

### DISCUSSION THREAD

**Discussion by megbrosseau-mcgill** [community/community#147090] [GitHub Education]
### Hi everyone,

Hi ! I have been approved for my Education submission a few days ago and I still haven't received my benefits and therefore can't use Copilot integrated in VS Code. 
Is it possible to give me access today ? I have work to do and therefore this is a bit time sensitive. 

Thabk you !

**Comment by davevad93** [comment]
Hi @megbrosseau-mcgill ,

GitHub no longer provide manual reviews, when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @megbrosseau-mcgill was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
